source/JABHandler.py

```diff
@@ -566,7 +566,7 @@ def getCurrentAccessibleValueFromContext(self):
 		return buf.value
 
 	def selectTextRange(self,start,end):

```

Comment:
Can you confirm these type annotations are correct?

```suggestion
	def selectTextRange(self, start: int, end: int) -> None:
```